### PR TITLE
YAML files are streams

### PR DESCRIPTION
**Pull request by ioggstream**
## This PR

- clarify that files are streams
- streams can contain one or more documents
- the specification is related to documents and reflects on streams.

***
<a href="https://pr-preview.s3.amazonaws.com/json-ld/yaml-ld/pull/49.html" title="Last updated on Jul 5, 2022, 7:15 PM UTC (9c192e5)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/json-ld/yaml-ld/49/1d2ffe5...9c192e5.html" title="Last updated on Jul 5, 2022, 7:15 PM UTC (9c192e5)">Diff</a>

### CODE REVIEW

File: spec/index.html [hunk 1]
```diff
@@ -423,5 +580,96 @@ <h3>Examples</h3>
...
   </section>
 
+  <section id="faq" class="informative" data-format="markdown">
+    <p class="ednote">REMOVE THIS SECTION BEFORE PUBLICATION.</p>
+
+    <h3>FAQ</h3>
+
+    #### Why does YAML-LD not preserve comments?
+    <p class="ednote">
+    [[JSON]] (and hence [[JSON-LD]]) does not support comments,
```
Comment (gkellogg) [inline spec/index.html:590]:
```suggestion
    [[JSON]] (and hence [[JSON-LD11]]) does not support comments,
```

File: spec/index.html [hunk 2]
```diff
@@ -283,9 +280,169 @@ <h2>Introduction</h2>
...
+      interoperability considerations.
+    </p>
+
+    <p>
+      This is because YAML is more flexible than JSON:
+    </p>
+
+      <ul>
+        <li>YAML supports different encodings, including UTF-8, UTF-16, and UTF-32.</li>
+        <li>YAML supports more data types than JSON.</li>
```
Comment (gkellogg) [inline spec/index.html:298]:
```suggestion
        <li>YAML supports more native data types than JSON.</li>
```

**Review by gkellogg — APPROVED** [review]
Looks great! A couple of suggestions inline.

**Comment by ioggstream** [comment]
@gkellogg PR should target branch ioggstream-8 :) so that it can be applied after merging 34 #34

**Pull request merged**